sdk/ownership/backend.go

```diff
@@ -73,12 +68,28 @@ func (s *Backend) CreateServiceOwnership(request cosmostypes.Request, serviceHas
 	return ownership, nil
 }
 
+// Delete deletes a ownership.
+func (s *Backend) Delete(req cosmostypes.Request, owner cosmostypes.AccAddress, resourceHash hash.Hash) error {
```

Comment:
owner is not used in this function.
it should either delete only the ownership of this specific owner or delete all the ownership associated to a resource. in this last case, remove owner from the parameter.